src/api/generatedTypes/AddEvent.ts

```diff
@@ -16,6 +16,7 @@ export interface AddEvent_addEvent_event_ticketSystem_InternalEventTicketSystem
 export interface AddEvent_addEvent_event_ticketSystem_TicketmasterEventTicketSystem {
   type: TicketSystem;
   url: string;
+  endTime: any | null;
```

Comment:
`any | null` does not look right... Does this type come as generated from the graphql introspection? API says 
```
class TicketmasterEventTicketSystem(ObjectType):
    child_password = graphene.String(child_id=graphene.ID())
    free_password_count = graphene.Int(required=True)
    used_password_count = graphene.Int(required=True)
    url = graphene.String(required=True)
    end_time = graphene.DateTime()
```

In graphql IDE it is marked as datetime as well
<img width="320" alt="image" src="https://user-images.githubusercontent.com/389204/205290702-0e0adba4-31f3-42e9-9b16-f8d43d1f3019.png">

Reply:
It seems like all the date times are `Any`. 🤷‍♂️ I had forget that.
![image](https://user-images.githubusercontent.com/389204/205290992-977ff828-df6f-4e76-a21e-d8359ca834e3.png)

Reply:
Yeah that is unfortunately the case.